app/controllers/users/confirmations_controller.rb

```diff
@@ -2,6 +2,12 @@ module Users
   class ConfirmationsController < Devise::ConfirmationsController
     include ValidEmailParameter
 
+    def create
+      super
+
+      UserMailer.already_confirmed(resource).deliver_later if resource.confirmed?
```

Comment:
I try to write code in a way that makes it readable and easy to understand without needing comments, but I can certainly add a comment here. In this case, if a resource is already confirmed and tries confirming again, they will receive an email that lets them know they already confirmed. I tried to name the methods in such a way that made that clear. What part did you find confusing?